apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -49,11 +53,34 @@ export default class SetupGuide extends React.Component {
         };
     }
   };
+
   render() {
     // Experiment 'microbit', displays Circuit Playground and Micro:Bit descriptions.
-    let isMicrobit = experiments.isEnabled('microbit');
+    const isMicrobit = experiments.isEnabled('microbit');
+    const chromeVersion = getChromeVersion();
+
     return (
       <div>
+        {isCodeOrgBrowser() && (
+          <Notification
```

Comment:
should we have two banners here? Is it possible to combine these into one message?

Reply:
I don't think anyone would see both...the first is for the Maker app, and the second for old chrome versions? Unless I'm misunderstanding something

Reply:
ah apologies, was trying to minimize screenshot screen real estate -- should have split those out!